src/oapv_port.h

```diff
@@ -180,11 +180,12 @@ void *oapv_memset_x128_avx(void* dst, int value, size_t size);
 
 #define oapv_mcpy(dst, src, size)    memcpy((dst), (src), (size))
 #define oapv_mset(dst, v, size)      memset((dst), (v), (size))
-#define oapv_mset_x64a(dst, v, size) memset((dst), (v), (size))
 #if X86_SSE
 #define oapv_mset_x128(dst, v, size) oapv_memset_x128_avx((dst), (v), (size))
+#define oapv_mset_x64a(dst, v, size) oapv_memset_x128_avx((dst), (v), (size))
```

Comment:
oapv_mset_x64a() doesn't guranttee 128byte-aligned size.
The AVX optimized oapv_mset_x128 doesn't seems to handle the unalinged size case.